changelog/new_add_three_state_boolean_cop.md

```diff
@@ -0,0 +1 @@
+* [#337](https://github.com/rubocop/rubocop-rails/issues/337): Add new `Rails/ThreeStateBoolean` cop. ([@fatkodima][])
```

Comment:
Sorry for delay. Can you use the new changelog addition way and update the PR number?

Reply:
> Can you use the new changelog addition way

Did not get, what should I use here?

> update the PR number

It references an original issue number. Should I change that?

Reply:
Oops! Totally wrong on my part. For some reason the mistook it for the PR number on CHANGELOG.md 💦 
I'm sorry for the confusion. You can leave it as is.